ui-manchette/src/hooks/isOverFlowed.ts

```diff
@@ -0,0 +1,39 @@
+import React, { useState, useLayoutEffect, useRef, RefObject } from 'react';
```

Comment:
`React` and `useRef` are not used
```suggestion
import { useState, useLayoutEffect, RefObject } from 'react';
```

Reply:
done